.circleci/continue-config.yml

```
@@ -981,7 +986,8 @@ workflows:
     jobs:
       ### ALL BRANCHES ###
 
-      - build
+      - build:
+          <<: *ignore-main
```

I think it is slightly surprising that this makes all the tests not run on main. I'm ok with solving it like this, if it works, but maybe we need a comment to explain what's going on.

```suggestion
          # We only want to run tests on branches (e.g. for the merge queue) and pull requests.
          # On main we do not need to re-run all the tests, as the merge queue guarantees that tests already passed.
          # For the edge release we declare the `build-edge` job below that only runs on main. 
          <<: *ignore-main
```

---

Makes sense and added your comment. I think this should make all the tests not run in main, since they all depend on `build` but hard to say if it works before merging.